<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    qproperty-alignment: 'AlignRight | AlignVCenter';
	border: 2px solid gray;
	background-color: white;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>xÂ²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: orange; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: orange; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: orange; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: lightblue; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: orange; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>âx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white; /* Default background color */
    qproperty-alignment: 'AlignCenter'; /* Align text to the center */
}

QPushButton:hover {
    background-color: lightgreen; /* Background color when hovered */
}

QPushButton:pressed {
    background-color: lightcoral; /* Background color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>BackSpace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
